components/trengo/actions/list-all-blocks/list-all-blocks.mjs

```diff
@@ -0,0 +1,31 @@
+import app from "../../trengo.app.mjs";
+
+export default {
+  key: "trengo-list-all-blocks",
+  name: "List All Blocks",
+  description: "List all blocks in a help center. [See the documentation](https://developers.trengo.com/reference/list-all-blocks)",
+  version: "0.0.1",
+  type: "action",
+  annotations: {
```

Comment:
This action should either return a paginated list, or provide a `page` prop for manual pagination.